packages/commonwealth/server/migrations/20230616214402-remove-duplicate-eth-chain-node.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkUpdate(
```

Comment:
Operations need to be transactionalized.

Reply:
Done